2009-02-13  Michael Tautschnig  <mt@debian.org>

	* conf/NFSROOT: Added git-core (thanks Sebastian Harl) (closes: #506459)
Index: trunk/conf/NFSROOT
===================================================================
--- trunk.orig/conf/NFSROOT
+++ trunk/conf/NFSROOT	
@@ -7,7 +7,7 @@
 dnsutils ntpdate dosfstools cvs jove xfsprogs xfsdump
 procinfo dialog discover mdetect
 console-tools console-common
-expect iproute udev subversion liblinux-lvm-perl
+expect iproute udev subversion liblinux-lvm-perl git-core
 
 # needed for the simple examples
 cfengine2 libapt-pkg-perl
